Resources/Locale/en-US/accent/german.ftl

```diff
@@ -0,0 +1,221 @@
+accent-german-words-1 = yes
+accent-german-words-1-2 = yea
+accent-german-words-replace-1 = ja
+
+accent-german-words-2 = no
+accent-german-words-replace-2 = nein
+
+accent-german-words-3 = the
+accent-german-words-replace-3 = ze
+
+accent-german-words-4 = shit
+accent-german-words-replace-4 = scheisse
+
+accent-german-words-5 = hospital
+accent-german-words-replace-5 = krankenhaus
+
+accent-german-words-6 = sausage
+accent-german-words-replace-6 = wurst
+
+accent-german-words-7 = sausages
+accent-german-words-replace-7 = wieners
+
+accent-german-words-8 = steak
+accent-german-words-replace-8 = schnitzel
+
+accent-german-words-9 = steaks
+accent-german-words-replace-9 = schnitzels
+
+accent-german-words-10 = beer
+accent-german-words-10-2 = beers
+accent-german-words-replace-10 = bier
+
+accent-german-words-11 = that was an order
+accent-german-words-replace-11 = das war ein Befehl
+
+accent-german-words-12 = butterfly
+accent-german-words-replace-12 = schmetterling
+
+accent-german-words-13 = my god
+accent-german-words-replace-13 = mein gott
+
+accent-german-words-14 = my
+accent-german-words-replace-14 = mein
+
+accent-german-words-15 = here
+accent-german-words-replace-15 = hier
+
+accent-german-words-16 = idiot
+accent-german-words-replace-16 = dummkopf
+
+accent-german-words-17 = idiots
+accent-german-words-replace-17 = dummköpfe
```

Comment:
remove this translation

Reply:
I think this one is perfectly fine to keep.